packages/@react-stately/toast/src/timer.ts

```diff
@@ -0,0 +1,21 @@
+export function Timer(callback, delay) {
```

Comment:
Do I need to add typescript definitions to this?

Reply:
eh, might as well. Prob can do it inline with callback being `() => void` maybe and `delay` being a `number`?